aggsender/types/types.go

```diff
@@ -61,6 +61,30 @@ type ChainGERReader interface {
 	GetInjectedGERsForRange(ctx context.Context, fromBlock, toBlock uint64) (map[uint64][]common.Hash, error)
 }
 
+// L1InfoTreeDataQuery is an interface defining functions that an L1InfoTreeDataQuery should implement
+// It is used to query data from the L1 Info tree
+type L1InfoTreeDataQuery interface {
+	// GetLatestFinalizedL1InfoRoot returns the latest processed l1 info tree root
+	// based on the latest finalized l1 block
+	GetLatestFinalizedL1InfoRoot(ctx context.Context) (*treetypes.Root, *l1infotreesync.L1InfoTreeLeaf, error)
+
+	// GetFinalizedL1InfoTreeData returns the L1 Info tree data for the last finalized processed block
+	// l1InfoTreeData is:
+	// - the leaf data of the highest index leaf on that block and root
+	// - merkle proof of given l1 info tree leaf
+	// - the root of the l1 info tree on that block
```

Comment:
Reorder to conform to the return values ordering:

```suggestion
	// - merkle proof of given l1 info tree leaf
	// - the leaf data of the highest index leaf on that block and root
	// - the root of the l1 info tree on that block
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/336/commits/b45bfd13336bb63c248f301b3e04bde29ebda5ab